plugins/resources/app/controllers/resources/quota_usage_controller.rb

```diff
@@ -8,7 +8,6 @@ class QuotaUsageController < ::Resources::ApplicationController
     QUOTA_RESOURCES = {
       compute: {service_type: :compute, resource_names: [:instances, :cores, :ram]},
       block_storage: {service_type: :volumev2, resource_names: [:volumes, :snapshots, :capacity]},
-      loadbalancing: {service_type: :network, resource_names: [:loadbalancers, :listeners, :pools, :l7policies]},
```

Comment:
@majewsky, @hgw77 have any side effects removing the old plugin and introducing the new one? The name of the plugin has changed.

Reply:
If I remember correctly, this is @andypf's code. But I don't know what side effect there should be.